descriptors/ModuleDescriptor-template.json

```diff
@@ -4,8 +4,18 @@
   "provides": [
     {
       "id": "circulation-bff-requests",
-      "version": "1.1",
+      "version": "1.2",
```

Comment:
We don't need a new version actually, we need a new interface (it's in my prev comment)